src/Link.js

```diff
@@ -98,11 +90,10 @@ export const Link = (
   )
 }
 
-Link.contextTypes = {
-  store: PropTypes.object.isRequired
-}
-
-const connector: Connector<OwnProps, Props> = connect()
+const mapState = state => ({
+  routesMap: selectLocationState(state).routesMap
+})
+const connector: Connector<OwnProps, Props> = connect(mapState)
```

Comment:
Shouldn't `dispatch` be mapped to props here?

Reply:
Hum, what do you have in mind?
Do you see a case dispatch is needed?

Reply:
It's already used and in a Redux context, dispatching action is needed, I guess. :)

Reply:
`dispatch` was not present before, is this necessary?
I wonder

Reply:
Seems not mandatory: https://stackoverflow.com/questions/34458261/how-to-get-simple-dispatch-from-this-props-using-connect-w-redux

Reply:
And this is [officially supported](https://react-redux.js.org/api/connect#mapdispatchtoprops-object-dispatch-ownprops-object), ok. 👍